checkstyle-tester/diff.groovy

```diff
@@ -60,6 +63,10 @@ def getCliOptions(args) {
             'Path to the patch checkstyle config file (required if baseConfig is specified)')
         c(longOpt: 'config', args: 1, required: false, argName: 'path', 'Path to the checkstyle ' \
             + 'config file (required if baseConfig and patchConfig are not secified)')
+        f(longOpt: 'failsOnError', required: false, 'Whether Maven Checkstyle Plugin should allow ' \
+            + 'errors to fail run (optional, default is false)')
+        g(longOpt: 'allowExcludes', required: false, 'Whether to allow excludes specified in the list of ' \
```

Comment:
I think it is better to reuse extraMvnOptions.
as introduction of arguments that works in one mode only is hard to describe in doc and use.
By extraMvnOptions we are flexible.

Reply:
Done.